src/networks/customized_loss/mustangs_loss.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import random
+from networks.customized_loss.heuristic_loss import HeuristicLoss
+from helpers.configuration_container import ConfigurationContainer
+
+
+class MustangsLoss(torch.nn.Module):
+
+    def __init__(self, applied_loss=None, name=None):
+        super(MustangsLoss, self).__init__()
+        self.losses_list = [torch.nn.BCELoss(), torch.nn.MSELoss(), HeuristicLoss()]
+
+        # We add a new parameter to network named 'randomized'. If it is set to always it selects randomly a new loss
+        # every minibatch in other case it selects randomly the loss function when the network is created
+        cc = ConfigurationContainer.instance()
+        if 'randomized' in cc.settings['network']:
+            self.option_randomized = 'always' in cc.settings['network']['randomized']
+            self.mustangs_always_random = self.option_randomized
```

Comment:
Rewrite with `self.mustangs_always_random = 'True' == cc.settings['network'].get('randomized', 'False')` to avoid the conditional statement?

Reply:
Sure